src/lib/spinner.cjs

```diff
@@ -1,17 +1,94 @@
 // @ts-check
-const logSymbols = require('log-symbols')
+const process = require('process')
+
+// eslint-disable-next-line no-restricted-modules
+const { Instance: ChalkInstance } = require('chalk')
 const ora = require('ora')
 
+const logSymbols = {
+  success: '🌈',
+  error: '🚫',
+}
+
+/**
+ * Chalk instance for CLI that can be initialized with no colors mode
+ * needed for json outputs where we don't want to have colors
+ * @param  {boolean} noColors - disable chalk colors
+ * @return {object} - default or custom chalk instance
+ */
+const safeChalk = function (noColors) {
+  if (noColors) {
+    const colorlessChalk = new ChalkInstance({ level: 0 })
+    return colorlessChalk
+  }
+  return new ChalkInstance()
+}
+/** The parsed process argv without the binary only arguments and flags */
+const argv = process.argv.slice(2)
+
+const chalk = safeChalk(argv.includes('--json'))
```

Comment:
We could simply use `chalk` from command-helpers here I think?

Reply:
I had some kind of warning in my terminal about not using safeChalk at this point, hence the copy paste. I'll look into it again.